draft-gpew-priv-ppm.md

```diff
@@ -772,13 +775,10 @@ and constructs its response to the aggregate request.
 
 #### Leader State
 
-The leader is required to issue aggregate requests in order, but reports are
-likely to arrive out-of-order. The leader SHOULD store reports for a time
-period proportional to the batch window before including them in an aggregate
-request. Failure to do so may result in out-of-order reports being dropped by
-the helper. The leader MUST NOT accept reports whose timestamps are in the
-future. Implementors MAY provide for some small leeway, usually no more than a
-few minutes, to account for clock skew.
+The leader is required to buffer reports while waiting to aggregate them. The
+leader MUST NOT accept reports whose timestamps are too far in the future.
```

Comment:
```suggestion
leader SHOULD NOT accept reports whose timestamps are too far in the future.
```

I don't see how we can enforce a MUST without a specific definition of "too far in the future."

Reply:
+1

Reply:
I agree, but the spec currently says MUST here, so this isn't something that's changed in this PR. Can we resolve this by leaving an OPEN ISSUE here?

Reply:
Why not just make the change here? It's obviously related to this PR.

Reply:
People often ask to keep spec PRs minimal, but I don't think anyone will object so I'll just change this. Done.